### DISCUSSION TITLE
Cross origin issue all of a sudden

### DISCUSSION THREAD

**Discussion by exander77** [chimurai/http-proxy-middleware#899] [Q&A]
### Checks

- [X] I understand project setup issues should be asked on [StackOverflow](https://stackoverflow.com/questions/tagged/http-proxy-middleware) or in [GitHub Discussions](https://github.com/chimurai/http-proxy-middleware/discussions).
- [X] I updated to latest `http-proxy-middleware`.

### Describe the bug (be clear and concise)

Suddenly I am getting:
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://support.apple.com/en-us/HT213526. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 403.
```
```
    createProxyMiddleware({
      target: 'https://api.darksky.net/',
      changeOrigin: true,
      pathRewrite: {'^/v1' : ''}
    })
```

### Step-by-step reproduction instructions

```shell
See above.
```

### Expected behavior (be clear and concise)

Proxy should work.

### How is http-proxy-middleware used in your project?

```shell
npm
```

### What http-proxy-middleware configuration are you using?

```typescript
See above.
```

### What OS/version and node/version are you seeing the problem?

```shell
Docker
```

**Comment by chimurai** [comment]
Sounds like a project setup/configuration issue.

Can't help without details about the problem.
Even better, provide a reproducible example to get better help
